services/core-web/src/components/admin/MinespaceUserList.js

```diff
@@ -33,7 +33,13 @@ const columns = [
     dataIndex: "mineNames",
     render: (text) => (
       <div title="Mines">
-        {text && text.map(({ mine_guid, mine_name }) => <span key={mine_guid}> {mine_name} </span>)}
+        {text &&
+          text.map(({ mine_guid, mine_name }) => (
+            <span key={mine_guid}>
+              {mine_name}
+              <br />
+            </span>
+          ))}
```

Comment:
Do we want to display a message if they have no mines? Or is this dealt with already?

Reply:
chatted IRL - there should never be a situation where there isn't at least 1 mine in this list.